### ISSUE TITLE
Allow closing parens or comma after closing curly brackets

### ISSUE THREAD

**Issue by krlmlr** [jimhester/lintr#113]
For `eval(bquote({...}))` and others.

**Comment by zachmayer** [comment]
:+1:

**Comment by JanSchulz** [comment]
IMO this should also not fail: `debug <- function(...){}`

**Comment by Enchufa2** [comment]
IMO something like this should not fail:

```r
out <- lapply(stuff, function(i) {
  do_something(i)
}) %>% unlist
```

**Comment by jimhester** [comment]
Yes I agree, but currently don't have time to look at this, pull requests welcome.

**Comment by Enchufa2** [comment]
@krlmlr, @zachmayer, @janschulz, @jimhester: #167 allows a closing parenthesis or a comma after a closing curly brace. Does any other case come to your mind before merging the pull request?
